Sources/ProjectSpec/TargetSource.swift

```diff
@@ -62,6 +72,14 @@ public struct TargetSource: Equatable {
         self.attributes = attributes
         self.resourceTags = resourceTags
     }
+    
+    private static func clean(path: String) -> String {
```

Comment:
Could [standardizingPath](https://developer.apple.com/documentation/foundation/nsstring/1407194-standardizingpath) be a better option than this method?